keystone.ts

```diff
@@ -40,6 +40,16 @@ export default withAuth(
         signed: { expiry: 5000 },
         forcePathStyle: true,
       },
+      my_S3_sounds: {
+        kind: "s3",
+        type: "file",
```

Comment:
Did you ever figure out what this `type` does and the significance of using `file` here instead of `image`, like above?

Reply:
As far as I could find, `image` is designated specifically for image files (I believe this means you can preview the images in the UI... I'm not sure what else the purpose is) while `file` works for pointing to files of any type